portfolio/src/main/java/com/google/sps/servlets/UserAuthServlet.java

```diff
@@ -48,21 +53,15 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         response.setContentType(TEXT_CONTENT_TYPE);
         response.getWriter().println(getWelcome());
         response.getWriter().println(getLoginLogoutLink());
-
-        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-        Query query = new Query("Comment").addSort("TIMESTAMP_PARAMETER", SortDirection.DESCENDING);
-        PreparedQuery results = datastore.prepare(query);
-        for (Entity entity : results.asIterable()) {
-            String text = (String) entity.getProperty("text");
-            String email = (String) entity.getProperty("email");
-
-        }
     }
 
     public String getWelcome() {
         String userNickname = "stranger";
         if (userService.isUserLoggedIn()) {
             userNickname = userService.getCurrentUser().getNickname();
+            if (userNickname.contains("@")){
+                return("<form method=\"POST\" action=\"/login\">Choose a username<input type = \"text\" name = \"nickname\"><input type = \"submit\" onclick=\"changeNickname()\"/></form>");
```

Comment:
Could this long string be stored as a variable? It is also way too long. Break up the into separate lines